deployments/docker/generate-config.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+import os
+from jinja2 import Environment, FileSystemLoader
+
+def env_override(default_value, key):
+    """ Gets value of the environment variable provided in the key """
+    return os.getenv(key, default_value)
+
+# Load templates into Jinja environment
+env = Environment(loader = FileSystemLoader('./config-templates'), trim_blocks=True, lstrip_blocks=True)
+env.filters['env_override'] = env_override
+
+# Renders config templates for each service into a config.yaml file
+services = ["provisioning", "verification", "vts"]
+for service in services:
+    template = env.get_template(service + '_tmpl.j2')
+    print(template.render())
+
+    file = open("../../" + service + "/cmd/" + service + "-service/config.yaml", 'w')
```

Comment:
I wonder whether this should be passed in as a parameter rather than being hard-coded?

Reply:
Which part should be passed in as a parameter? Is it the file directory (line 20) or the template directory (line 18)?

Reply:
I'd pass in the whole path, without assuming any specific structure, and let the caller figure that out.